content/blog/_posts/inclusive-infra/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Towards More Inclusive Infrastructure"
+date: 2021-04-14
+slug: inclusive-infra
+authors: Reinhardt Quelle
+---
+Posted by Reinhardt Quelle, Dir. Site Reliability Engineering for Cisco Emerging Technolgies and Incubation
```

Comment:
Would this make more sense in the 'authors' front matter?

/me wonders how we are rendering authors on blog posts currently...

Reply:
which led me to dhttps://github.com/inclusivenaming/website/pull/49